v1.1/known-limitations.md

```diff
@@ -63,7 +111,7 @@ To detect if this is happening, open the [Admin UI](admin-ui-access-and-navigate
 
 For example, let's say that latency is 10ms from nodes in datacenter A to nodes in datacenter B but is 100ms from nodes in datacenter A to nodes in datacenter C. To ensure A's and B's relative proximity is factored into lease holder rebalancing, you could restart the nodes in datacenter A and B with a common region, `--locality=region=foo,datacenter=a` and `--locality=region=foo,datacenter=b`, while restarting nodes in datacenter C with a different region, `--locality=region=bar,datacenter=c`.
 
-## Roundtrip to `STRING` does not respect precedences of `:::` and `-`
+### Roundtrip to `STRING` does not respect precedences of `:::` and `-`
```

Comment:
I think this was fixed: cockroachdb/cockroach#15617

Reply:
Great. Updated in the 1.0 limitations and removed from the 1.1 limitations.